02-basics-2/50-selected-meetup/SelectedMeetupApp.js

```diff
@@ -1,75 +1,61 @@
-import { defineComponent } from 'vue'
-// import { getMeetup } from './meetupsService.ts'
+import { defineComponent, ref, onMounted, watch } from 'vue'
+import { getMeetup } from './meetupsService.ts'
 
 export default defineComponent({
   name: 'SelectedMeetupApp',
 
-  setup() {},
+  setup() {
+    const checkedId = ref(1)
+    let meetupTitle = ref('')
+
+    const findMeetUpTitle = async () => {
+      const { title } = await getMeetup(checkedId.value)
+      return meetupTitle.value = title
+    }
+
+    onMounted(async () => {
+      await findMeetUpTitle()
+    })
+
+    watch(checkedId, async (newId, oldId) => {
+      if (newId !== oldId) {
```

Comment:
This condition is redundant. Vue doesn't count it as a change, if an old value is equal to the new value (there is no value change).

Reply:
Got it!